fastlane.gemspec

```diff
@@ -102,7 +102,7 @@ Gem::Specification.new do |spec|
 
   # Development only
   spec.add_development_dependency('rake')
-  spec.add_development_dependency('rspec', '~> 3.9.0')
+  spec.add_development_dependency('rspec', '~> 3.10.0')
```

Comment:
Do you actually need this bump in "this PR"? If so can you please clarify that?

Reply:
Yes, to get the `include(...).once` matched.